src/Resources/config/fos_http_cache.yml

```diff
@@ -13,4 +16,4 @@ user_context:
 tags:
     # Configure tag header for FosHttpCache
     # See docs/using_tags.md for further info on usage
-    header: '%ezplatform.http_cache.tags.header%'
+    response_header: '%ezplatform.http_cache.tags.header%'
```

Comment:
Todo: Drop custom header, same with `user_hash_header: X-User-Hash` once we have BC for old header being sent in responses.